examples/ppo_softprompt_sentiments.py

```diff
@@ -0,0 +1,45 @@
+from typing import List
+
+import torch
+from torch import nn
+from transformers import pipeline
+
+import wandb
+from trlx.data.configs import TRLConfig
+from trlx.model.accelerate_ppo_model import AcceleratePPOModel
+from trlx.model.accelerate_ppo_softprompt_model import AcceleratePPOSoftpromptModel
+from trlx.orchestrator.ppo_orchestrator import PPOOrchestrator
+from trlx.pipeline.ppo_pipeline import PPOPipeline
+from trlx.utils.loading import get_model, get_orchestrator, get_pipeline
+
+
+if __name__ == "__main__":
+    cfg = TRLConfig.load_yaml("configs/ppo_softprompt_config.yml")
+
+    sentiment_pipe = pipeline(
+        "sentiment-analysis", "lvwerra/distilbert-imdb", device=-1
+    )
+
+    def reward_fn(samples: List[str]):
+        sent_kwargs = {
+            "return_all_scores": True,
+            "function_to_apply": None,
+            "batch_size": cfg.method.chunk_size,
+        }
+        pipe_outputs = sentiment_pipe(samples, **sent_kwargs)
+        scores = torch.tensor([output[1]["score"] for output in pipe_outputs])
+        return scores
+
+    model: AcceleratePPOSoftpromptModel = get_model(cfg.model.model_type)(cfg)
```

Comment:
You don't need this. Just remove the type case from ppo_sentiments and leave the imports. The registery (e.g. get_model) will be able to determine which model type to load based off the config you use.